pkg/controller/servicediscovery/servicediscovery_controller.go

```diff
@@ -35,8 +35,8 @@ var log = logf.Log.WithName("controller_servicediscovery")
 
 const (
 	componentName                 = "submariner-lighthouse"
-	deploymentName                = "submariner-lighthouse-agent"
-	lighthouseCoreDNSName         = "submariner-lighthouse-coredns"
+	DeploymentName                = "submariner-lighthouse-agent"
```

Comment:
Please un-do all those changes which are not necessary now for the patch.

To grab older versions of the files you can use

git checkout HEAD~1 pkg/controller/servicediscovery/servicediscovery_controller.go
...
...
etc...
then when you do `git status` you will see those files are modified, commit them...
and the changes will be removed from your patch.

Note HEAD~1 assumes that your PR has only one patch, so it will grab the state of the file before your patch.